en/docs/tutorials/scenarios/scenario4-user-signup-approval-flow.md

```diff
@@ -1,4 +1,72 @@
-# Scenario 4 - User Signup Approval Flow
+# Scenario 4 - Signing up a new user
+
+## User Story
+
+**Quantis allows external users to access their APIs. To take the full benefit out of these APIs, they have decided to open the registration to outside users since it can be a burden to enter all the details for every single user/consumer. Therefore, they request an approval workflow for user registration in such a way that the users will be registered to the system when a user with administrative privileges approves their registration after a manual validation of the user.  By allowing external parties to register to their system under a supervision, Quantis expects higher revenue in future. **
+
+**_Time to Complete : 3 mins_**
+
+![Signup description]({{base_path}}/assets/img/tutorials/scenarios/signup-desc.png)
+
+WSO2 API Manager provides extension points to trigger workflow tasks for many operations such as Application creation, subscription creation, user signup, etc. Out of the box WSO2 API Manager ships with simple approval workflow. 
+
+This demo setup already comes with self-signup and signup workflow enabled for the Qantis tenant domain. If you need to check how to configure this check [enabling self-signup](https://apim.docs.wso2.com/en/latest/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup/) and [enabling signup workflow](https://apim.docs.wso2.com/en/latest/develop/customizations/adding-a-user-signup-workflow/) documentation. 
```

Comment:
Please don't hard code links. Add the relative path to these files where possible.

Reply:
Fixed